devops/docker/.env

```diff
@@ -0,0 +1,3 @@
+#VUE_APP_FOLDER_ID=<>
+ESIMMON_PASSWORD=itsasecret
+ADMIN_PASSWORD=letmein
```

Comment:
I think we should force the user to pass these on the command line so they change them?